examples/chart/teleport-cluster/values.yaml

```diff
@@ -288,12 +321,31 @@ gcp:
   # You can override this to a blank value if the worker node running Teleport already has a service account which grants access.
   credentialSecretName: teleport-gcp-credentials
 
-# Settings for high availability. These are not used in "standalone" mode.
-# When using "custom" mode, you must use highly-available storage (etcd, DynamoDB or Firestore) for multiple replicas to be supported.
-# Manually configuring NFS-based storage or ReadWriteMany volume claims is NOT supported and will result in errors.
+
+# `highAvailability` contains settings controlling how Teleport pods are
+# replicated and scheduled. This allows to run Teleport in a highly-available
```

Comment:
As above, either "This allows ____ to run..." or something like "This allows Teleport to run..."